ball.ai/src/main/java/com/google/sps/data/Team.java

```diff
@@ -0,0 +1,27 @@
+package com.google.sps.data;
+
+import java.lang.reflect.Field;
+import java.util.List;
+
+public class Team {
+  public String team_name, team_nickname, team_abbrev, division_id, conference;
+
+  public Team(String city, String name, String abbrv, String division, String conference) {
+    this.team_name = city;
+    this.team_nickname = name;
+    this.team_abbrev = abbrv;
+    this.division_id = division;
+    this.conference = conference;
```

Comment:
You might want to add a field for team ID, since the NBA seems to have teh useful codes that we can use for unambiguously identifying the teams